gpu/install_gpu_driver.sh

```diff
@@ -223,6 +223,34 @@ EOF
   systemctl --now enable gpu-utilization-agent.service
 }
 
+function config_gpu_exclusive_mode {
+  # check if running spark3, if not, enable GPU exclusive mode
+  spark-submit --version &> /tmp/version.txt
+  if ! grep -q "version 3" "/tmp/version.txt" ; then
+    # include exclusive mode on GPU
+    nvidia-smi -c EXCLUSIVE_PROCESS
+  fi
+}
+
+function config_gpu_isolation {
+  # download GPU discovery script
+  mkdir -p /usr/lib/spark/scripts/gpu/
+  cd /usr/lib/spark/scripts/gpu/
+  wget https://raw.githubusercontent.com/apache/spark/master/examples/src/main/scripts/getGpusResources.sh
```

Comment:
Remove `cd` command and use `curl` with retry options and destination option instead:
```bash
local -r gpu_resources_url=https://raw.githubusercontent.com/apache/spark/master/examples/src/main/scripts/getGpusResources.sh
curl -fsSL --retry-connrefused --retry 10 --retry-max-time 30 \
    "${gpu_resources_url}" -o /usr/lib/spark/scripts/gpu/getGpusResources.sh
```

Reply:
Made the change following suggestion